build.env

```diff
@@ -39,6 +39,7 @@ HELM_VERSION=v3.9.2
 
 # minikube settings
 MINIKUBE_VERSION=v1.26.1
+MINIKUBE_ISO_URL=https://storage.googleapis.com/minikube-builds/iso/14783/minikube-v1.26.1-1661377864-14783-amd64.iso
```

Comment:
If this doesn't get updated and CI works fine, am good with this one.

Reply:
I think @Madhu-1 concern is vaild, can we copy this to some local file server @nixpanic (I remember we supplied a native link with NBD patches before) ?

Reply:
We don't really want to host ISO images ourselves. If this link stays available until a next minikube version contains the fscrypt feature, I'd prefer to using this.